src/cascadia/TerminalSettingsEditor/ProfileViewModel.h

```diff
@@ -26,6 +27,28 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         Editor::ProfileViewModel _Profile{ nullptr };
     };
 
+    struct BellSoundViewModel : BellSoundViewModelT<BellSoundViewModel>, ViewModelHelper<BellSoundViewModel>
+    {
+    public:
+        BellSoundViewModel() = default;
+        BellSoundViewModel(hstring path) :
+            _Path{ path }
+        {
+            PropertyChanged([this](auto&&, auto&& e) {
+                if (e.PropertyName() == L"Path")
+                {
+                    _NotifyChanges(L"DisplayPath", L"SubText", L"FileExists", L"ShowSubText");
+                }
+            });

```

Comment:
Similar to the other comment, is there anything changing `Path` anymore at runtime?